jinahub/rankers/CatboostRanker/catboost_ranker.py

```diff
@@ -0,0 +1,168 @@
+__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+from typing import Dict, List, Optional
+
+import numpy as np
+from catboost import CatBoostRanker, Pool
+
+from jina import DocumentArray, Executor, requests
+from jina.logging.logger import JinaLogger
+
+
+class CatboostRanker(Executor):
+    def __init__(
+        self,
+        query_features: List[str],
+        match_features: List[str],
+        label: str,
```

Comment:
I would call it `relevance_label`

Reply:
i think lablel could be anything user want to set? such as `relevance`, or `groundtruth` ..

Reply:
no, groundtruth label wont make sense. The label is a relevance label for the prediction